fontbe/src/kern.rs

```diff
@@ -9,55 +9,91 @@ use fea_rs::{
 use fontdrasil::{
     coords::NormalizedLocation,
     orchestration::{Access, AccessBuilder, Work},
+    types::GlyphName,
 };
 use fontir::{
-    ir::{KernPair, KernParticipant},
+    ir::{GlyphOrder, KernPair, KernParticipant},
     orchestration::WorkId as FeWorkId,
 };
-use ordered_float::OrderedFloat;
+use log::debug;
```

Comment:
Just as a general observation but I prefer not to import these macros by name, and instead just call them as `log::debug!("my message")`. The main rationale for this is simply that it makes it easier to grep for all the places where you're logging. A secondary rationale is that there's a (loose) convention in Rust to avoid importing functions directly, and instead import their parent module.. I can't find a great source for this, but it's [mentioned here by klabnik](https://github.com/rust-lang/style-team/issues/24#issuecomment-252288601). I like this convention but am not zealous about it.

Reply:
I think we do this relatively consistently; maybe file an issue and we'll fix it eveyrwhere plus add a presubmit?